packages/store/test/StoreCore.t.sol

```diff
@@ -11,7 +11,7 @@ import { FieldLayout, FieldLayoutLib } from "../src/FieldLayout.sol";
 import { Schema } from "../src/Schema.sol";
 import { PackedCounter, PackedCounterLib } from "../src/PackedCounter.sol";
 import { StoreMock } from "../test/StoreMock.sol";
-import { IStoreErrors } from "../src/IStoreErrors.sol";
+import { Store_TableAlreadyExists, Store_InvalidSplice, Store_IndexOutOfBounds, Store_InvalidBounds, Store_TableNotFound, Store_InvalidStaticDataLength, Store_InvalidValueSchemaStaticLength, Store_InvalidResourceType, Store_InvalidValueSchemaDynamicLength, Store_InvalidFieldNamesLength, Store_InvalidKeyNamesLength, Store_InvalidValueSchemaLength } from "../src/StoreErrors.sol";
```

Comment:
is it important to be explicit about imports when we're in a test?

wondering about:
```solidity
import "../src/StoreErrors.sol"
```

Reply:
Good shout, would clean up all these imports!